### PR TITLE
FreeBSD compatibility patch

### PR DESCRIPTION
**Pull request by yurivict**

### CODE REVIEW

File: folly/Subprocess.cpp [hunk 1]
```diff
@@ -24,6 +24,7 @@
 #include <sys/prctl.h>
 #endif
 #include <fcntl.h>
+#include <unistd.h>
```
Comment (Orvid) [inline folly/Subprocess.cpp]: This is already included via the `folly/portability/Unistd.h` include further down. Remove this include.

File: folly/portability/Stdlib.cpp [hunk 2]
```diff
@@ -147,6 +147,10 @@ int unsetenv(const char* name) {
 #include <string>
 #include <vector>
 
+#if defined(__FreeBSD__) || defined(__DragonFly__)
```
Comment (Orvid) [inline folly/portability/Stdlib.cpp]: Is there really nowhere in the libc used by freebsd that declares `environ`?

**Comment by yurivict** [comment]
> Is there really nowhere in the libc used by freebsd that declares environ?

The man page doesn't mention  any headers:
```
ENVIRON(7)         FreeBSD Miscellaneous Information Manual         ENVIRON(7)

NAME
     environ — user environment

SYNOPSIS
     extern char **environ;
```
This may be an oversight, but it is this way currently.

-------------------------

I'll update the pull request for the other things.

**Comment by yurivict** [comment]
Overcome by events.

**Pull request closed without merging**